src/main/webapp/index-script.js

```diff
@@ -74,10 +73,10 @@ function searchByCoordinates(coordinate) {
   });
 }
 
-// Search using text query
-function searchByText(textQuery){
+/* Search Places API for relevant locations using text query. */
+function searchByText(textQuery) {
   // Create the places service.
-
+  const service = new google.maps.places.PlacesService(map);
```

Comment:
Creating const service not necessary because create var service only 5 lines below this one